specs/android/send_firebase_push_notification.yaml

```diff
@@ -0,0 +1,29 @@
+---
+name: Send a firebase push notification to local Android emulator
+command: adb shell am broadcast \
+  -n {{your_app_package_name}}/com.google.firebase.iid.FirebaseInstanceIdReceiver \
+  -a "com.google.android.c2dm.intent.RECEIVE" \
+           --es "title" "{{notification_title}}" \
+           --es "body" "{{notification_body}}" \
+           --es "deeplink" "{{deeplink}}"
+tags:
+  - android
+  - adb
+  - firbase
```

Comment:
nit: I think there's a typo here with firebase

Reply:
Good catch! Should be fixed in 9b44064 😄 Thanks!